### ISSUE TITLE
Learning rate calculation of adam wrong?

### ISSUE THREAD

**Issue by YurongYou** [steveKapturowski/tensorflow-rl#20]
https://github.com/steveKapturowski/tensorflow-rl/blob/bcc9b2a966d7e5dee53f2a1fef1d5c355af85a79/algorithms/actor_learner.py#L319

**Comment by steveKapturowski** [comment]
Hi @YurongYou, I modified the code so I'm not changing the value of opt_st.lr.value. It appears to be functioning as expected as of commit 728f510a84681d49631e1a72294673e7950172ed

**Comment by YurongYou** [comment]
Great!
I have to say your code is so well written! I learnt a lot from it. Thank you so much!

**Comment by steveKapturowski** [comment]
No problem! Glad you've found it useful
